cypress/support/fragments/data_import/dataImport.js

```diff
@@ -431,6 +431,16 @@ export default {
       });
   },
 
+  verifyTrashIconInvisibleForUser: (user) => {
+    cy.login(user.username, user.password, {
```

Comment:
Login and navigation actions does not related to DataImport module. I'd recommend to move these steps to test and leave only verification of button absense here.

Reply:
updated